### ISSUE TITLE
Error in C# options when opening project

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#4356]
The following error happens in the terminal when i open a project:

```
ERROR [2018-03-28 12:26:05Z]: An unhandled exception has occured. Terminating MonoDevelop? False
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.CSharp.DocumentContextExtension+<GetOptionsAsync>d__0.MoveNext () [0x000fa] in /Users/mikayla/code/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp/DocumentContextExtension.cs:47 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:152 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x00037] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:187 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:156 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:128 
  at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:357 
  at MonoDevelop.CSharp.Formatting.CSharpTextEditorIndentation+<HandleTextOptionsChanged>d__13.MoveNext () [0x00055] in /Users/mikayla/code/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp.Formatting/CSharpTextEditorIndentation.cs:141 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:152 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/AsyncMethodBuilder.cs:1018 
  at MonoDevelop.Ide.DispatchService+GtkSynchronizationContext+TimeoutProxy.HandlerInternal (System.IntPtr data) [0x00016] in /Users/mikayla/code/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DispatchService.cs:77 
```

> VS bug [#591635](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/591635)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.5.0.994 (d15-7)

Pull Request #4366 merged by: Mike Krüger
Author: mono
Commit: df3f7b658073ac17306dfc7847dac867125c0bd7 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.5.0.994 (d15-7)

Pull Request #4366 merged by: Mike Krüger
Author: mono
Commit: df3f7b658073ac17306dfc7847dac867125c0bd7 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.359 (master)

Pull Request #4363 merged by: Mike Krüger
Author: mono
Commit: cd312ed2c78f7b5e4120e8f18bf8cf10c72a0aa8 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.359 (master)

Pull Request #4363 merged by: Mike Krüger
Author: mono
Commit: cd312ed2c78f7b5e4120e8f18bf8cf10c72a0aa8 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #4363 merged by: Mike Kr??ger
Author: mono
Commit: cd312ed2c78f7b5e4120e8f18bf8cf10c72a0aa8 (xamarin/monodevelop)
